sphinx_rtd_theme/versions.html

```diff
@@ -1,12 +1,12 @@
 {% if READTHEDOCS %}
 {# Add rst-badge after rst-versions for small badge style. #}
   <div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="{{ _('Versions') }}">
-    <span class="rst-current-version" data-toggle="rst-current-version">
+    <button class="rst-current-version" aria-expanded="false" aria-controls="docs_versions" data-toggle="rst-current-version">
       <span class="fa fa-book"> Read the Docs</span>
       v: {{ current_version }}
       <span class="fa fa-caret-down"></span>
-    </span>
-    <div class="rst-other-versions">
+    </button>
+    <div class="rst-other-versions" id="docs_versions">
```

Comment:
These changes need to be repeated in https://github.com/readthedocs/sphinx_rtd_theme/blob/master/docs/_templates/layout.html#L11